integration/swarm/swarm.bats

```diff
@@ -66,6 +70,29 @@ setup() {
 	done
 }
 
+@test "obtain hostname of the replicas" {
+	# this will help to obtain the hostname of
+	# the replicas from the curl
+	unset http_proxy
```

Comment:
It might be best if this test saved all the proxy variable and then restored them at the end.